samples/react-native/getting-started/README.md

```diff
@@ -0,0 +1,41 @@
+# Getting Started
+
+`getting-started` is a simple app that allows you to get a 1:1 chat app quickly up and running
+through the PubNub Network. It features a simple chat window with a message input field. After
+starting the app, you can send sample messages as a default `myFirstUser` by typing them in an input
+field and messages will pile up on the screen as you send them.
+
+![Getting Started app for React Native](screenshot.png)
+
+## Prerequisites
+
+This application uses [React v18.0.0](https://www.npmjs.com/package/react/v/18.0.0),
+[React Native v0.69.5](https://www.npmjs.com/package/react-native),
+[PubNub JavaScript SDK v7.2.0](https://www.pubnub.com/docs/sdks/javascript/), and
+[PubNub React SDK v3.0.1](https://www.pubnub.com/docs/chat/react/setup).
+
+To use the app, you need:
+
+- [npm](https://docs.npmjs.com/cli/v6/commands/npm-install) or
+  [yarn](https://yarnpkg.com/getting-started/install)
+- [Node.js](https://nodejs.org/en/download/)
```

Comment:
@piotr-suwala should we specify the exact version it will work with? We already know it doesn't work with node 18.